bridges/darwinia-ethereum/bridge/src/service/check.rs

```diff
@@ -86,29 +86,26 @@ async fn run(tracker: &Tracker) -> color_eyre::Result<()> {
         if txs.is_empty() {
             tracing::info!(
                 target: "darwinia-ethereum",
-                "[ethereum] All transactions checked"
+                "[ethereum] [check] All transactions checked"
             );
             tokio::time::sleep(std::time::Duration::from_secs(task_config.interval_check)).await;
             continue;
         }
-
         let tx = txs.get(0).unwrap();
 
-        let verified = match helpers::is_verified(&darwinia, tx).await {
+        let tx_hash = array_bytes::hex2bytes(&tx.block_hash).map_err(|_e| {
+            BridgerError::Hex(format!(
+                "Failed to convert hex({}) to bytes.",
+                &tx.block_hash
+            ))
+        })?;
+        let tx_index = tx.tx_index;
+        let verified = match client.ethereum().is_verified(&tx_hash, tx_index).await {
             Ok(v) => v,
             Err(e) => {
-                if let Some(substrate_subxt::Error::Rpc(_)) =
-                    e.downcast_ref::<substrate_subxt::Error>()
-                {
-                    return Err(e);
-                }
-                let err_msg = format!("{:?}", e).to_lowercase();
-                if err_msg.contains("restart") {
```

Comment:
To double check

Reply:
https://github.com/darwinia-network/bridger/pull/434/commits/db46c9252044bb031fc4ccb169e7216ba12131b4